lib/immediate-juttle-job.js

```diff
@@ -20,6 +20,9 @@ class ImmediateJuttleJob extends JuttleJob {
 
         // Any warnings for this program.
         self._warnings = [];
+
+        // Any debug logs for this program.
+        self._logs = [];
```

Comment:
Is this just debug level logs, or all logs when debug is true? I think the latter, right? If so the comment is kinda misleading.